### ISSUE TITLE
[Feature Request] Display Bells to Poki Conversion rates

### ISSUE THREAD

**Issue by DraconicNEO** [averne/Turnips#56]
I think it would be a very good idea for this tool to have the ability to display the Bell to Poki Conversion rates for each day of the week with a graph similar to what we have for turnips. That way we can know when the conversion rates are highest between bells to Poki and vice versa.

**Comment by averne** [comment]
Hi, I don't own the DLC and I'm completely disinterested in the game.
If you want it, please implement it (and the other features you requested) yourself.

**Comment by DraconicNEO** [comment]
> Hi, I don't own the DLC and I'm completely disinterested in the game. 

Oh I see. Well it would probably be best to update the readme to state that the app isn't under active development anymore.

>If you want it, please implement it (and the other features you requested) yourself.

Not sure if this is was intended to be hostile but it feels slightly hostile. I'm sorry if the ~~requests~~ suggestions I made are annoying. Now that I know that the app isn't being developed anymore and aren't wanted I won't make any more. My apologies if I annoyed you.
